logic/src/commonMain/kotlin/com/wire/kalium/logic/data/id/QualifiedId.kt

```diff
@@ -1,8 +1,16 @@
 package com.wire.kalium.logic.data.id
 
+import com.wire.kalium.network.tools.KtxSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+@Serializable
 data class QualifiedID(
     val value: String,
     val domain: String
-)
-
-typealias ConversationId = QualifiedID
+) {
+    fun toJson(): String = Json.encodeToString(this)
```

Comment:
```suggestion
    fun toJson(): String = KtxSerializer.json.encodeToString(this)
```

Better to use the same serializer to encode and to decode.